Packs/ExabeamDataLake/Integrations/ExabeamDataLake/ExabeamDataLake.py

```diff
@@ -0,0 +1,273 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+""" CONSTANTS """
+
+HEADERS = {"Accept": "application/json", "Csrf-Token": "nocheck"}
+
+ISO_8601_FORMAT = "%Y-%m-%d"
+
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Exabeam DataLake integration. Overrides BaseClient
+    """
+
+    def __init__(self, base_url: str, username: str, password: str, verify: bool,
+                 proxy: bool, headers):
+        super().__init__(base_url=f'{base_url}', headers=headers, verify=False, proxy=proxy, timeout=20)
+        self.username = username
+        self.password = password
+
+        self._login()
+
+    def _login(self):
+        """
+        Logs in to the Exabeam API using the provided username and password.
+        This function must be called before any other API calls.
+        Note: the session is automatically closed in BaseClient's __del__
```

Comment:
Are you sure that the __del__ in the BaseClient closes the session in the same manner as calling exabeam end point for logout off session as disscussed in the kickoff?